play.py

```diff
@@ -405,14 +405,14 @@ def play_aidungeon_2():
                     if load_ID.startswith("gs://"):
                         story_manager.cloud = True
                         load_ID = load_ID[5:]
-                    if story_manager.has_encryption():
+                    result = story_manager.load_from_storage(load_ID)
+                    if result is None:

```

Comment:
previously only asked for the password if encryption is already enabled; now it will first try to load with the current configuration, including any encryption, and only if that fails ask for a new password. This has the benefit of not requiring the user to type in passwords for loading saves within the same game, and also letting the user enter a password when loading an encrypted file if encryption is currently off.